### PR TITLE
Add `$.balanceTextUpdate()` method

### PR DESCRIPTION
**Pull request by rileyjshaw**
Adds a `balanceTextUpdate()` method, initially discussed in https://github.com/adobe-webplatform/balance-text/issues/56. This is already called on load and resize, but it's nice to expose it for other events that we're not accounting for.

Depends on https://github.com/adobe-webplatform/balance-text/pull/60.

### CODE REVIEW

**Comment by rileyjshaw** [comment]
(Updated to match #60)

File: README.md [hunk 1]
```diff
@@ -60,3 +60,4 @@ Code is minified using: http://marijnhaverbeke.nl/uglifyjs
 * v 1.3.x - Debounce resizing events, more accurate space width estimate
 * v 1.4.0 - Add support for nested tags (rileyjshaw)
 * v 1.5.0 - Re-balance text on resize for manually triggered selectors (rileyjshaw)
+* v 1.6.0 - Add balanceTextUpdate() method (rileyjshaw)
```
Comment (redmunds) [inline README.md]: Can you also add a paragraph + code sample in README.md explaining usage of new API call?

**Comment by rileyjshaw** [comment]
Done!

**Comment by redmunds** [comment]
Looks good. Merging.

**Pull request merged**